src/neo/Trie/MPT/Helper.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+
+namespace Neo.Trie.MPT
+{
+    public static class Helper
+    {
+        public static byte[] Concat(this byte[] a, byte[] b)
```

Comment:
We should reuse the methods and move to `neo/Helper.cs` these most generics one

https://github.com/neo-project/neo/blob/4755824fece8c577874910fd69702e38ed777193/src/neo/Helper.cs#L38

Reply:
ok